### ISSUE TITLE
Setting buffer size to 50ms leads to truncated mbrola-fr4 output

### ISSUE THREAD

**Issue by sthibaul** [espeak-ng/espeak-ng#772]
Hello,

With the default parameters,
```espeak-ng -v mb-fr4 Bonjour```
properly speaks the provided word. But with a buffer size set to 50ms, the sound is completely cut. That can be reproduced by doing the following change, to make `espeak-ng` explicitly request 50ms buffer:

```
diff --git a/src/espeak-ng.c b/src/espeak-ng.c
index 1634cc1d..5ce3f55c 100644
--- a/src/espeak-ng.c
+++ b/src/espeak-ng.c
@@ -600,7 +600,7 @@ int main(int argc, char **argv)
                }
        } else {
                // play the sound output
-               result = espeak_ng_InitializeOutput(PLAYBACK_MODE, 0, devicename[0] ? devicename : NULL);
+               result = espeak_ng_InitializeOutput(PLAYBACK_MODE, 50, devicename[0] ? devicename : NULL);
                samplerate = espeak_ng_GetSampleRate();
        }
 
```

```espeak-ng -v mb-fr4 Bonjour```

will almost not pronounce anything. Passing a longer sentence gets more output, but still really not much, it seems there is a bogus computation somewhere there.

This was noticed with espeak-ng 1.49 and 1.50, with the mb-fr4 voice only, not the integrated fr voice.

**Comment by valdisvi** [comment]
@sthibaul, why you need such short buffer?
Most of phoneme sounds are around 100ms length (and long vowels are 300ms), therefore with such short buffer even one phoneme will not fit into it.
[here](https://github.com/espeak-ng/espeak-ng/blob/d32b32caf24c02e82f231b51df3694df60504503/src/libespeak-ng/speech.c#L279) it shows, that default (probably sane minimum) is 60ms.
If you deal with some latency issue, probably it is longer than 50ms and source of it is somewhere else.

**Comment by sthibaul** [comment]
> why you need such short buffer?

I don't really, it's just that espeakup happened to be passing over 50 there, and that was breaking mbrola voices. espeak-ng should be defensive about this, you never know what value a program would want to try to pass.

> default (probably sane minimum) is 60ms.

Then the passed value should be clamped to this minimum, if smaller values won't work.

**Comment by valdisvi** [comment]
@rhdunn, then maybe code should be changed to:
```
if (buffer_length <= 60)
		buffer_length = 60;
```
with some comments about settings?

**Comment by rhdunn** [comment]
That makes sense.

**Comment by valdisvi** [comment]
@rhdunn, I created pull request #775 of the fix for your review.

**Comment by sthibaul** [comment]
So this was fixed by #775
